src/super_gradients/common/registry/registry.py

```diff
@@ -59,3 +61,5 @@ def decorator(cls: Callable) -> Callable:
 register_unet_backbone_stage = create_register_decorator(registry=BACKBONE_STAGES)
 register_unet_up_block = create_register_decorator(registry=UP_FUSE_BLOCKS)
 register_target_generator = create_register_decorator(registry=ALL_TARGET_GENERATORS)
+register_collate_function = create_register_decorator(registry=ALL_COLLATE_FUNCTIONS)
```

Comment:
Registering is essential but not sufficient...see my summary.

Reply:
updated in commit "updated_ fixes"